ClientSideEncryptionToCPKN/ClientSideKeyVaultKeyToCustomerManagedKey/README.md

```diff
@@ -0,0 +1,29 @@
+# Key Vault Client Side Encryption to Customer Managed Key Server Side Encryption
```

Comment:
One thing that's missing in this Readme is step by step instruction how to run this with sample data.

something like:
1. go to <that directory>
2. run "dotnet build setup"
3. run setup
4. run something else...
5. observe output